packages/expo-sms/android/src/main/java/expo/modules/sms/SMSModule.java

```diff
@@ -51,24 +52,13 @@ public void setModuleRegistry(ModuleRegistry moduleRegistry) {
 
   @ExpoMethod
   public void sendSMSAsync(final ArrayList<String> addresses, final String message, final Promise promise) {
-    if (!getContext().getPackageManager().hasSystemFeature(PackageManager.FEATURE_TELEPHONY) &&
-        !getContext().getPackageManager().hasSystemFeature(PackageManager.FEATURE_TELEPHONY_CDMA)) {
-      promise.reject(ERROR_TAG + "_UNAVAILABLE", "SMS service not available");
-      return;
-    }
-
-    if (mPromise != null) {
+    if (mPendingPromise != null) {
```

Comment:
Could you please also add CHANGELOG entry for the bug you fixed and that promise won't be rejected if "there's no telephony feature" on the device?